### ISSUE TITLE
Adding a preset with numbers only is allowed but causes a TypeError: expected string or buffer

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/framework#1633]
```
May 09 14:33:13 ctrl-221-1 celery[18621]: 2017-05-09 14:33:13 90100 +0200 - ctrl-221-1 - 18621/139969431508736 - celery/celery.worker.job - 1291 - ERROR - Task alba.add_preset[2dbe9113-85f1-4b65-abd6-b9608cb9e71b] raised unexpected: TypeError('expected string or buffer',)
May 09 14:33:13 ctrl-221-1 celery[18621]: Traceback (most recent call last):
May 09 14:33:13 ctrl-221-1 celery[18621]:   File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
May 09 14:33:13 ctrl-221-1 celery[18621]:     R = retval = fun(*args, **kwargs)
May 09 14:33:13 ctrl-221-1 celery[18621]:   File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
May 09 14:33:13 ctrl-221-1 celery[18621]:     return self.run(*args, **kwargs)
May 09 14:33:13 ctrl-221-1 celery[18621]:   File "/opt/OpenvStorage/ovs/lib/albapreset.py", line 60, in add_preset
May 09 14:33:13 ctrl-221-1 celery[18621]:     if not re.match(Toolbox.regex_preset, name):
May 09 14:33:13 ctrl-221-1 celery[18621]:   File "/usr/lib/python2.7/re.py", line 141, in match
May 09 14:33:13 ctrl-221-1 celery[18621]:     return _compile(pattern, flags).match(string)
May 09 14:33:13 ctrl-221-1 celery[18621]: TypeError: expected string or buffer
```

![screen shot 2017-05-09 at 14 33 14](https://cloud.githubusercontent.com/assets/8393135/25851112/bd80a490-34c4-11e7-88fc-1f93b4ce078d.png)

**Comment by khenderick** [comment]
The API does auto parameter resolving, so it sees the value is a number, so it converts it to a number.

**Comment by wimpers** [comment]
Fix by disabling giving a number only name or if alba allows, change the API  parameter resolving.

**Comment by khenderick** [comment]
Fixed by openvstorage/framework-alba-plugin#419 packaged in openvstorage-backend-1.9.2-dev.1497255983.0bc7ac9

**Comment by JeffreyDevloo** [comment]
Packaged in openvstorage-backend_1.9.2-1_amd64.deb
